pkg/controller/instance/instance_controller.go

```diff
@@ -440,7 +441,7 @@ func pipesMap(planName string, plan *v1beta1.Plan, tasks []v1beta1.Task, emeta *
 
 // startPlanExecution mark plan as to be executed
 // this modifies the instance.Status as well as instance.Metadata.Annotation (to save snapshot if needed)
-func startPlanExecution(i *v1beta1.Instance, planName string, ov *v1beta1.OperatorVersion) error {
+func startPlanExecution(i *v1beta1.Instance, planName string, ov *v1beta1.OperatorVersion, currentTime time.Time) error {
```

Comment:
Let's not have `currentTime` as a parameter, but set it in the function instead, similar to `UpdateInstanceStatus`.